css-view-transitions-1/Overview.bs

```diff
@@ -197,40 +197,55 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 # Pseudo-elements # {#pseudo}
 
-	The following <dfn export>view-transition pseudo-elements</dfn> represent the various items being animated.
+## Pseudo-element root ## {#pseudo-root}
 
-	The ''::view-transition'' pseudo-element acts as a grouping element for other [=view-transition pseudo-elements=]
-	and has the document's [=document element=] as its [=originating element=].
+	Issue: This definition should be moved to [[css-pseudo-4]].
 
-	Note: For example, '':root::view-transition'' selector matches this pseudo-element,
-	but ''div::view-transition'' does not.
+	A <dfn>pseudo-element root</dfn> is a type of [=tree-abiding pseudo-element=] that is the [=tree/root=] in a [=tree=] of [=tree-abiding pseudo-elements=],
```

Comment:
So `::view-transition` will be the pseudo-element root. Correct? Can we add some prose to clarify how this tree attaches to the DOM. Rough idea below:

"The ancestor of a pseudo-element root is a [node](https://dom.spec.whatwg.org/#node). This node is the root's originating element and the [ultimate originating element](https://www.w3.org/TR/selectors-4/#ultimate-originating-element) for all pseudo-elements in this tree."

Reply:
Actually reading further I'm not sure if pseudo-element root is limited to view-transition or any pseudo-element which has descendants. Can you confirm.

Reply:
`::view-transition` is the only root. I don't mind adding that extra paragraph, but the originating element for `::view-transition` is already defined later in the spec.